validator/api/src/main/java/tech/pegasys/teku/validator/api/FileBackedGraffitiProvider.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.api;
+
+import java.nio.file.Path;
+import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+
+/**
+ * A type to hold the current graffiti value. Graffiti can be loaded from 2 places:
+ *
+ * <ul>
+ *   <li>The config file at initialisation
```

Comment:
(nit) This package shouldn't care about details around config files and CLI arguments - those are higher-level concerns that might change
```suggestion
 *   <li>The default value supplied at initialisation
```